app/routes/assessments/get-challenge.js

```diff
@@ -0,0 +1,25 @@
+import Ember from 'ember';
+import RSVP from 'rsvp';
+
+export default Ember.Route.extend({
+
+  model(params) {
+    const store = this.get('store');
+    return RSVP.hash({
+      assessment: store.findRecord('assessment', params.assessment_id),
+      challenge: store.findRecord('challenge', params.challenge_id)
+    });
+  },
+
+  /*
+   * This method is called when transitionTo is called with a context in order to populate the URL.
+   * – http://emberjs.com/api/classes/Ember.Route.html#method_serialize
+   */
```

Comment:
suggestion: add `XXX` in the beginning of the comment to mark it as “this is a surprising behavior”